### ISSUE TITLE
Addon no longer functional in v2.1.0

### ISSUE THREAD

**Issue by steenert** [SanjoSolutions/CommodityBuyerAndSeller#2]
Hi,

The addon is prompting to sell items, but after pressing the confirm button it does not actually post anything. it keeps asking to post with no result.

**Comment by SanjoSolutions** [comment]
I have released [2.1.1](https://github.com/SanjoSolutions/CommodityBuyerAndSeller/releases/tag/v2.1.1) which hopefully fixes this.

**Comment by fatmagic** [comment]
Hey Sanjo - jump on this issue, as I'm not sure it's fixed. Running /run CommodityBuyerAndSellerData.doConfigured() is not firing it up. Here is my CommodityBuyerAndSellerData.lua - which was working on 2.0.0:

```
CommodityBuyerAndSellerData = {}

function CommodityBuyerAndSellerData.doConfigured()
	CommodityBuyerAndSeller.sell(193243, 1000, 30, 200)
	CommodityBuyerAndSeller.sell(193244, 1000, 30, 550)
endaq
```

Prices on AH are above the minimums for both items. Just doesn't do anything when I fire off the /run command.
Also double checked the ID's on WoW Head - both are correct for the items in my bag.
